include/seqan3/alignment/band/static_band.hpp

```diff
@@ -26,9 +29,6 @@ namespace seqan3
 class static_band
```

Comment:
Actually the class should be deprecated. 

```suggestion
class SEQAN3_DEPRECATED_310 static_band
```

Reply:
And then it would need an updated documentation that this class has been deprecated and one should use the align_cfg::band_fixed_size instead.